src/state/users/actions.js

```diff
@@ -16,3 +22,35 @@ export const fetchUsers = (page = 1, params) => dispatch =>
     dispatch(setUsers(data.payload));
     dispatch(setPage(data.metaData));
   });
+
+
+export const fetchUserForm = username => dispatch =>
+  getUser(username).then((response) => {
+    if (response.ok) {
+      response.json().then((json) => {
```

Comment:
You would need to return the response.json() promise in order to keep the promise chained:
`return response.json().then((json) => {`

BUT since we adopted the convention to return a `new Promise()` in each new API and explicitly resolving it, you'd better use that for this API too (as you did in `sendPutUser`)